### ISSUE TITLE
Having difficulty setting up appliance as standalone database server

### ISSUE THREAD

**Issue by goforgold** [ManageIQ/manageiq-appliance_console#129]
Hi Guyz

Sorry, I am a bit new to ManageIQ. I am trying to do a POC which includes 2 nodes (1 webserver + 1 database server).

I used Vagrant and VirtualBox to deploy two separate machines and trying to set up one of these machines as a standalone database server. The release I am onto is ivanchuk-7.

I am following the below steps:

1. Stop the EVM process from Appliance Console
2. Rest Database from 5) Configure Application -> 4) Reset Configured Database
3. Create Internal database from 5) Configure Application and then option 1.
4. I fill the database password it asks for and then the following output appears

```
Activating the configuration using the following settings...
Host:     localhost
Username: root
Database: vmdb_production

An internal database already exists.
Choose "Reset Configured Database" to reset the existing installation
Configuration failed

Press any key to continue.
```

I can't get past this error. What is the correct way to set up the appliance as a standalone database server? Please help.

**Comment by Fryguy** [comment]
I don't think you need to do step 3, because, by default, there is already an internal database, which is why it's asking you to just reset the internal database.  @bdunne can you help out here?

**Comment by bdunne** [comment]
On the first appliance, you shouldn't need to do any database configuration since it's initialized on first boot of the appliance.  On the second appliance, you would want to use 5) Configure Application, 3) Join Region in External Database, then enter the connection information for the first appliance with the database.

**Comment by goforgold** [comment]
Thanks @bdunne. In that case, I think I can do what I am trying to do here. Thanks for your help.

I was trying to follow [this article](https://www.manageiq.org/docs/reference/latest/high_availability_guide/) step-by-step. We might need to update the documentation or add the version information the docs are applied to. I can do that if you point to the repo containing the docs part.

> 5. In the appliance console, configure the following:
>     1. Configure the hostname by selecting Set Hostname.
>     2. Select Configure Database.
>     3. Select Create key to create the encryption key. You can create a new key, or use an existing key on your system by selecting Fetch key from remote machine and following the prompts.
>     4. Select Create Internal Database.
>     5. Select the database disk. ManageIQ then activates the configuration.
>     6. For Should this appliance run as a standalone database server?, select y. Selecting this option configures this appliance as a database-only appliance, and therefore the MIQ application and evmserverd processes will not run. This is required in highly available database deployments.
> This configuration is not reversible.
>     7. Create the database password.

**Comment by bdunne** [comment]
Yes, the documentation for this looks outdated.  We added the first boot database initialization script a while ago.  The components of the document live [here](https://github.com/ManageIQ/manageiq-documentation/tree/master/high_availability_guide/_topics).  Any updates would be much appreciated.

**Comment by Fryguy** [comment]
@agrare I think this one goes away when we change the appliance builds.

**Comment by tjyang** [comment]
> @agrare I think this one goes away when we change the appliance builds.

Default disk layout set to be DB node is the prefer way to me. Since one can easily remove lv_pg and turn into a worker node. Turn a miq worker node into miq DB node is require more work.

**Comment by Fryguy** [comment]
@agrare Can this be closed now?

**Comment by agrare** [comment]
👍 yes
